### PR TITLE
docs: amend code of conduct

### PR DESCRIPTION
**Pull request by Rumperuu**
&lt;opinion&gt;
Explicit enumeration makes sense in legal documents, but CoCs are
more aspirational so I don't think it makes sense in this context.
You will end up either having to constantly add new things to the
list, or declare that the list is non-exhaustive (in which there's
not much point in enumerating in the first place).

This is doubly true given the inherently woolly realm of appropriate
interpersonal interactions that a CoC is implemented to moderate.

Also, remember the KISS principle. Personally, I think this is the
pinnacle of CoC design: https://github.com/karlgroves/dontbeadick
However, I'm sure there is a slightly-more-professional middle
ground ;)
&lt;/opinion&gt;

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -1,40 +1,23 @@
...
+All participants in this community must abide by the following policy:
 
-* Demonstrating empathy and kindness toward other people
-* Being respectful of differing opinions, viewpoints, and experiences
-* Giving and gracefully accepting constructive feedback
-* Accepting responsibility and apologizing to those affected by our mistakes,
-  and learning from the experience
-* Focusing on what is best not just for us as individuals, but for the
-  overall community
+> Be excellent to each other
```
Comment (lolzim) [inline CODE_OF_CONDUCT.md:15]: Why this '>' sign?

**Review by lolzim — APPROVED** [review]
I put a comment inline just to 1) try this & 2) highlight  something I found weird.

Otherwise, KISS here too, great.

**Review by markcheret — APPROVED** [review]
I'm happy with this. Let's adopt this for now and let it evolve over time.

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (markcheret) [reply CODE_OF_CONDUCT.md:15]: IMO this is to let it visually stand out.

Comment (Rumperuu) [reply CODE_OF_CONDUCT.md:15]: It's Markdown syntax for a blockquote:

Screenshot from 2021-04-28 07-10-31

Comment (lolzim) [reply CODE_OF_CONDUCT.md:15]: ok

**Pull request merged**